server/app/lib/utils/fusor/openstack_dns.rb

```diff
@@ -9,7 +9,27 @@ def initialize(openstack_deployment)
       end
 
       def compare
-        get_overcloud_dns == satellite_dns && get_undercloud_dns == satellite_dns
+        overcloud_dns == satellite_dns && undercloud_dns == satellite_dns
+      end
+
+      def overcloud_dns
+        neutron = Fog::Network::OpenStack.new(undercloud)
+        neutron.subnets.first.dns_nameservers.first
+      end
+
+      def undercloud_dns
+        client = ::Utils::Fusor::SSHConnection.new(@openstack_deployment.undercloud_ip_address,
+                                                   @openstack_deployment.undercloud_ssh_username,
+                                                   @openstack_deployment.undercloud_ssh_password)
+        io = StringIO.new
```

Comment:
Perhaps force pushing amended commits wasn't the best idea with the new review system... but I added an ensure as per your comment